timeline/templates/timeline/_sort_header.html

```diff
@@ -0,0 +1,21 @@
+<div class="card sodar-sort-card">
```

Comment:
There's a slight bug here: we are including a card within a card, resulting in two cards being displayed on top of each other in the layout.